packages/daemon/__tests__/integration/scenarios/balances.test.ts

```diff
@@ -0,0 +1,201 @@
+/**
```

Comment:
suggestion: Maybe we should have this test (the only one so far) on the `integration` folder and keep utilities, scenario configs and docker configuration on folders that indicate whats inside them.

what do you think?

Reply:
I agree, done in f94a27c9243a70c4ecf40c2602425ff6e93210c9